cli/template/base/README.md

```diff
@@ -55,7 +55,7 @@ Please note that Next.js requires a different process for buildtime (available i
      });
    ```
 
-2. Remove the `env`-import from [next.config.mjs](./next.config.mjs):
+2. Remove the `env`-import from [next.config.mjs](./template/base/next.config.mjs):
```

Comment:
This link isn't correct?

Reply:
Sorry for all the suggestions, but when I think of it, we could merge steps 1 and 2? Using diff-mode makes that quite handy, what's a think?